themes/default/content/registry/packages/azure-static-website/installation-configuration.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Azure Static Website Installation
+meta_desc: Information on how to install the Azure Static Website component.
+layout: installation
+---
+
+## Installation
+
+The Pulumi Azure Static Website component is available as a package in all Pulumi languages:
+
+* JavaScript/TypeScript: [`@pulumi/azure-static-website`](https://www.npmjs.com/package/@pulumi/azure-static-website)
+* Python: [`pulumi_azure_static_website`](https://pypi.org/project/pulumi-azure-static-website/)
+* Go: [`github.com/pulumi/pulumi-azure-static-website/sdk`](https://pkg.go.dev/github.com/pulumi/pulumi-azure-static-website/sdk)
+* .NET: [`Pulumi.AzureStaticWebsite`](https://www.nuget.org/packages/Pulumi.AzureStaticWebsite/)
+
+If you are new to Pulumi and do not yet have our Pulumi installed, see our [getting started](https://www.pulumi.com/docs/get-started/) guide to get up and running.
```

Comment:
This extra "our" has been copied around a bit -- I've been meaning to get in there and clear it out, but haven't had time. 😅  

```suggestion
If you are new to Pulumi and do not yet have Pulumi installed, see our [getting started](https://www.pulumi.com/docs/get-started/) guide to get up and running.
```